apps/prairielearn/src/batched-migrations/20250822211536_enrollments__joined_at__backfill.sql

```diff
@@ -0,0 +1,14 @@
+-- BLOCK select_bounds
+SELECT
+  MAX(id)
+FROM
+  enrollments;
+
+-- BLOCK update_enrollments_joined_at
+UPDATE enrollments
+SET
+  joined_at = created_at
+WHERE
+  joined_at IS NULL
```

Comment:
Nit, likely shouldn't make a difference but can't hurt and should avoid write churn for old rows.

```suggestion
  joined_at IS NULL
  AND created_at IS NOT NULL
```